Client/src/Components/Label/index.jsx

```diff
@@ -116,70 +116,93 @@ ColoredLabel.propTypes = {
  * @param {Object} props
  * @param { 'up' | 'down' | 'cannot resolve'} props.status - The status for the label
  * @param {string} props.text - The text of the label
+ * @param {boolean} props.hasDot - Whether to show a dot or not
  * @returns {JSX.Element}
  * @example
  * // Render an active label
  * <StatusLabel status="up" text="Active" />
  */
 
-const StatusLabel = ({ status, text, customStyles }) => {
-	const theme = useTheme();
-	const colors = {
-		up: {
-			dotColor: theme.palette.success.main,
-			bgColor: theme.palette.success.bg,
-			borderColor: theme.palette.success.light,
-		},
-		down: {
-			dotColor: theme.palette.error.text,
-			bgColor: theme.palette.error.bg,
-			borderColor: theme.palette.error.light,
-		},
-		paused: {
-			dotColor: theme.palette.warning.main,
-			bgColor: theme.palette.warning.bg,
-			borderColor: theme.palette.warning.light,
-		},
-		pending: {
-			dotColor: theme.palette.warning.main,
-			bgColor: theme.palette.warning.bg,
-			borderColor: theme.palette.warning.light,
-		},
-		"cannot resolve": {
-			dotColor: theme.palette.unresolved.main,
-			bgColor: theme.palette.unresolved.bg,
-			borderColor: theme.palette.unresolved.light,
-		},
-	};
-
-	// Look up the color for the status
-	const { borderColor, bgColor, dotColor } = colors[status];
-
-	return (
-		<BaseLabel
-			label={text}
-			styles={{
-				color: dotColor,
-				backgroundColor: bgColor,
-				borderColor: borderColor,
-				...customStyles,
-			}}
-		>
-			<Box
-				width={7}
-				height={7}
-				bgcolor={dotColor}
-				borderRadius="50%"
-				marginRight="5px"
-			/>
-		</BaseLabel>
-	);
+const StatusLabel = ({ status, text, hasDot = false, customStyles }) => {
+  const theme = useTheme();
+  const colors = {
+    up: {
+      dotColor: theme.palette.success.main,
+      bgColor: theme.palette.success.bg,
+      borderColor: theme.palette.success.light,
+    },
+    down: {
+      dotColor: theme.palette.error.text,
+      bgColor: theme.palette.error.bg,
+      borderColor: theme.palette.error.light,
+    },
+    paused: {
+      dotColor: theme.palette.warning.main,
+      bgColor: theme.palette.warning.bg,
+      borderColor: theme.palette.warning.light,
+    },
+    pending: {
+      dotColor: theme.palette.warning.main,
+      bgColor: theme.palette.warning.bg,
+      borderColor: theme.palette.warning.light,
+    },
+    400: {
+      dotColor: theme.palette.warning.main,
+      bgColor: theme.palette.warning.bg,
+      borderColor: theme.palette.warning.light,
+    },
+    500: {
+      dotColor: theme.palette.error.text,
+      bgColor: theme.palette.error.bg,
+      borderColor: theme.palette.error.light,
+    },
```

Comment:
At this moment, I think it’s better to generalize the color codes to 400 and 500 to avoid dealing with a large number of codes and colors. In the future, we can consider making adjustments.